### PR TITLE
feature/AOAF-546-MUDdatebugfix

### PR DESCRIPTION
**Pull request by myadavch**

### CODE REVIEW

File: src/main/java/uk/gov/companieshouse/accounts/filing/mapper/FilingGeneratorMapper.java [hunk 1]
```diff
@@ -30,7 +31,12 @@ public FilingApi mapToFilingApi(AccountsFilingEntry accountsFilingEntry) {
         Map<String, String> descriptionValue = Collections.singletonMap("made up date", madeUpDate);
 
         var filingApiEntity = new FilingApi();
-        filingApiEntity.setDescription("Package accounts made up to " + formatMadeUpDate(madeUpDate));
+        if(PackageTypeApi.OVERSEAS == accountsFilingEntry.getPackageType()){
+            filingApiEntity.setDescription("Package accounts with package type overseas");
+        }else{
+            filingApiEntity.setDescription("Package accounts made up to " + formatMadeUpDate(madeUpDate));
+        }
```
Comment (amartin7663) [inline src/main/java/uk/gov/companieshouse/accounts/filing/mapper/FilingGeneratorMapper.java:38]: Could we fix the spacing here a bit
```suggestion
        if (PackageTypeApi.OVERSEAS == accountsFilingEntry.getPackageType()) {
            filingApiEntity.setDescription("Package accounts with package type overseas");
        } else {
            filingApiEntity.setDescription("Package accounts made up to " + formatMadeUpDate(madeUpDate));
        }
```

**Review by myadavch — COMMENTED** [review]
accepted the Alex m suggestion

**Review by amartin7663 — APPROVED** [review]

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Pull request merged**